android/CMakeLists.txt

```diff
@@ -26,8 +26,10 @@ if(${IS_REANIMATED_EXAMPLE_APP})
     string(APPEND CMAKE_CXX_FLAGS " -DIS_REANIMATED_EXAMPLE_APP")
 endif()
 
-if(${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+if(${CMAKE_BUILD_TYPE} MATCHES "Debug")
```

Comment:
What's the motivation behind this change?

Reply:
> Specifies the build type on single-configuration generators (e.g. [Makefile Generators](https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html#makefile-generators) or [Ninja](https://cmake.org/cmake/help/latest/generator/Ninja.html#generator:Ninja)). Typical values include Debug, Release, RelWithDebInfo and MinSizeRel, but custom build types can also be defined.

https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html

`CMAKE_BUILD_TYPE` also supports custom values so I thought let's support everything that contains "Debug".

Reply:
:sus: